### PR TITLE
Norman

### PR DESCRIPTION
**Pull request by georgeadrila**

### CODE REVIEW

File: app/Config/Routes.php [hunk 1]
```diff
@@ -34,6 +34,15 @@
...
+$routes->get('contract', 'Home::Contract');
+$routes->get('payement', 'Home::Payement');
+$routes->get('designation', 'Home::Designation');
+$routes->get('leave', 'Home::Leave');
+$routes->get('loan', 'Home::Loan');
+$routes->get('shift', 'Home::Shift');
+$routes->get('permission', 'Home::Permission');
 
 
 /*
```
Comment (aaron-oluk) [inline app/Config/Routes.php:48]: added page routes

File: app/Controllers/Home.php [hunk 2]
```diff
@@ -27,13 +25,123 @@ public function CreateEmployee()
...
+            'head' => "Permissions",
+            'link' => "Asign Permission"
+        ];
+        $data = [
+            'active' => 'active-list',
+        ];
+        return view('permission', array_merge($data, $title));
+    }
+        
 }
```
Comment (aaron-oluk) [inline app/Controllers/Home.php:147]: rewrite functions to fetch data and work on permissions

File: app/Views/branch-master.php [hunk 3]
```diff
@@ -4,45 +4,43 @@
...
-                        <li><a href="#">Contract Type</a>
+                        <li><a href="<?= base_url('permission'); ?>">Permissions</a>
                         <i class='bx bxs-chevron-right arrow'></i></li>
-                        <li><a href="#">Permissions</a>
-                        <i class='bx bxs-chevron-right arrow'></i></li>
-                        <li><a href="#">Job Type</a>
+                        <li><a href="<?= base_url('job'); ?>">Job Type</a>
                         <i class='bx bxs-chevron-right arrow'></i></li>
                     </ul>
                 </div>
```
Comment (aaron-oluk) [inline app/Views/branch-master.php:46]: removing empty links and assigning fully functional pages to them

File: app/Views/contract.php [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+            </div>
+            <div class="modal-footer">
+                <!-- <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button> -->
+                <button type="button" class="btn btn-primary">Add</button>
+            </div>
+            </div>
+        </div>
+     </div>
+
+<?= $this->endSection() ?>
```
Comment (aaron-oluk) [inline app/Views/contract.php:117]: must recheck codebase

File: app/Views/create-employee.php [hunk 5]
```diff
@@ -203,13 +199,8 @@
...
-                                    <option selected>Select Permissions</option>
-                                    <option value="1">One</option>
-                                    <option value="2">Two</option>
-                                    <option value="3">Three</option>
-                                </select>
+                                <label for="Upload">Upload Employee DOC<span>*</span></label>
+                                <input type="file" class="form-control form-control-sm" id="inputGroupFile01">
                             </div>
 
                             <div class="col">
```
Comment (aaron-oluk) [inline app/Views/create-employee.php:206]: reasigning empty links

File: app/Views/deductions.php [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+            </div>
+            <div class="modal-footer">
+                <!-- <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button> -->
+                <button type="button" class="btn btn-primary">Add</button>
+            </div>
+            </div>
+        </div>
+     </div>
+
+<?= $this->endSection() ?>
```
Comment (aaron-oluk) [inline app/Views/deductions.php:117]: deductions page worked on and code updated

File: app/Views/department.php [hunk 7]
```diff
@@ -94,5 +94,36 @@
...
+            </div>
+            <div class="modal-footer">
+                <!-- <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button> -->
+                <button type="button" class="btn btn-primary">Add</button>
+            </div>
+            </div>
+        </div>
+     </div>
 
 <?= $this->endSection() ?>
```
Comment (aaron-oluk) [inline app/Views/department.php:129]: taking care of the different sidebar options

**Review by aaron-oluk — APPROVED** [review]
I approve this code to go on production, checked its viability

**Review by aaron-oluk — COMMENTED** [review]
nice changes and would recommend all code to be checked before it's published

File: app/Controllers/Home.php (hunk 2 above)
Comment (normanmukama) [reply app/Controllers/Home.php:147]: Am not understanding well what you are trying to say

**Pull request merged**